virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -24,6 +24,37 @@
 LOG = logging.getLogger('avocado.' + __name__)
 
 
+def get_new_disk(session):
+    """
+    Identifies the single disk below /dev/
+    that's not mounted currently.
+
+    :param session: VM session
+    :param test: Test instance
```

Comment:
here there is no test parameter, others LGTM

Reply:
Thanks for spotting it, fixed.